image-layout.md

```diff
@@ -134,7 +134,7 @@ The `imageLayoutVersion` value will align with the OCI Image Specification versi
 
 ### oci-layout Example
 
-```json
+```json,title=Image%20Layout&valid=enabling&mediatype=oci%2Dlayout
```

Comment:
I'm in favor of adding a title, but “Image Layout” is the whole structure, not just the `oci-layout` file.  Can we follow the section headers and call this “oci-layout file” or some such?

What is `valid=enabling` about?

I don't think we want to invent a one-off media type here.  If we want a media type for this file format (e.g. for example validation), I think we should declare a media type for the schema like we do with our other schemas (e.g. [here][1] and [here][2] for manifests).

[1]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc4/manifest.md#L9
[2]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc4/media-types.md#L7